package.json

```diff
@@ -63,7 +63,9 @@
   },
   "devDependencies": {
     "@octokit/rest": "^18.0.0",
+    "@tsconfig/node10": "^1.0.7",
```

Comment:
A whole dependency for something that seems as simple as the following seems rather overkill to me:
```
{
  "compilerOptions": {
    "lib": ["es2018"],
    "module": "commonjs",
    "target": "es2018",
    "strict": true,
    "esModuleInterop": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true
  }  
}
```

Reply:
it seems like this is some sort of TS best practice 🤷